_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+				},
+			};
+		},
+	},
+
+	sitemaps: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Sitemaps' ),
+				details: __( 'Automatically generate sitemaps for all your content.' ),
+				checked: getSetting( state, 'sitemaps' ),
+				optionsLink: '#/settings?term=sitemaps',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sitemaps: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	sharing: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Sharing' ),
+				details: __( 'Increase sharing of your posts and pages. ' ),
+				checked: getSetting( state, 'sharedaddy' ),
+				optionsLink: '#/settings?term=sharedaddy',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sharedaddy: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	shortcodes: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Shortcode Embeds' ),
+				details: __( 'Embed YouTube videos, and other content easily.' ),
+				checked: getSetting( state, 'shortcodes' ),
+				optionsLink: '#/traffic?term=embeds',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { shortcodes: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	shortlinks: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'WP.me Shortlinks' ),
+				details: __( 'Build quick links for sharing.' ),
+				checked: getSetting( state, 'shortlinks' ),
+				optionsLink: '#/settings?term=shortlink',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { shortlinks: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'simple-payments-block': {
+		mapStateToProps: state => {
+			const sitePlan = getSitePlan( state );
+			const planClass = getPlanClass( sitePlan.product_slug );
+
+			const inCurrentPlan = [ 'is-premium-plan', 'is-business-plan' ].includes( planClass );
+
+			let upgradeLink;
+			if ( ! inCurrentPlan ) {
+				upgradeLink = '#/plans';
+			}
+
+			let info;
+			let configureLink;
+			if ( inCurrentPlan ) {
+				info = __( `Included with ${ sitePlan.product_name }` );
+				configureLink = 'https://jetpack.com/support/simple-payment-button/';
+			}
+
+			return {
+				title: __( 'Simple Payments Block' ),
+				details: __( 'A simple way to accept payments.' ),
+				checked: inCurrentPlan,
+				isDisabled: inCurrentPlan,
+				isPaid: true,
+				configureLink,
+				upgradeLink,
+				info,
+			};
+		},
+	},
+
+	'site-accelerator': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Site Accelerator' ),
+				details: __( 'Enable for faster images and a faster site.' ),
+				checked: getSetting( state, 'photon-cdn' ),
```

Comment:
Site Accelerator is currently a combination of 2 modules: `photon` and `photon-cdn`. Maybe we should toggle both at once.

Reply:
Good call, updated to do this.